Velaptor/InternalExtensionMethods.cs

```diff
@@ -29,6 +29,39 @@ namespace Velaptor
     /// </summary>
     internal static class InternalExtensionMethods
     {
+        /// <summary>
+        /// Determines whether this string instance does not start with the specified character.
+        /// </summary>
+        /// <param name="stringToCheck">The string to check.</param>
+        /// <param name="value">The character to compare.</param>
+        /// <returns><c>true</c> if <paramref name="value"/> matches the beginning of this string; otherwise, <c>false</c>.</returns>
+        public static bool DoesNotStartWith(this string stringToCheck, char value) => !stringToCheck.StartsWith(value);
+
+        /// <summary>
+        /// Determines whether this string instance does not start with the specified string.
```

Comment:
SS
```suggestion
Determines whether or not this string instance starts with the specified character.
```

Reply:
FYI - I left the word "string" at the end unchanged.  This is because this method does not check for characters, it checks for strings.